lib/spack/spack/cmd/ci.py

```diff
@@ -163,6 +165,8 @@ def ci_reindex(args):
 
     buildcache.update_index(remote_mirror_url, update_keys=True)
 
+    return 0
```

Comment:
I think the `return 0`'s can be removed -- as `None` is equivalent as far as the command dispatch logic is concerned.